sql-plugin/src/main/scala/org/apache/spark/sql/hive/rapids/GpuHiveSimpleUDF.scala

```diff
@@ -18,23 +18,22 @@ package org.apache.spark.sql.hive.rapids
 
 import ai.rapids.cudf.{NvtxColor, NvtxRange}
 import com.nvidia.spark.RapidsUDF
-import com.nvidia.spark.rapids.{ExprChecks, ExprMeta, ExprRule, GpuColumnVector, GpuExpression, GpuOverrides, GpuScalar, RepeatingParamCheck, TypeSig}
+import com.nvidia.spark.rapids.{GpuColumnVector, GpuExpression, GpuScalar}
 import com.nvidia.spark.rapids.RapidsPluginImplicits._
 import org.apache.hadoop.hive.ql.exec.UDF
+import org.apache.hadoop.hive.ql.udf.generic.GenericUDF
 
 import org.apache.spark.sql.catalyst.expressions.{Expression, UserDefinedExpression}
 import org.apache.spark.sql.hive.HiveShim.HiveFunctionWrapper
-import org.apache.spark.sql.hive.HiveSimpleUDF
 import org.apache.spark.sql.types.DataType
 import org.apache.spark.sql.vectorized.ColumnarBatch
 
-/** GPU-accelerated version of Spark's `HiveSimpleUDF` */
-case class GpuHiveSimpleUDF(
+abstract class GpuHiveUDFBase(
```

Comment:
nit: Should we rename this file now that it is for more than just `GpuHiveSimpleUDF`?

Reply:
I renamed it to `hiveUDFs` which is the same name used for a similar file in Apache Spark.